src/test/ui/resolve/issue-22692.stderr

```diff
@@ -1,11 +1,34 @@
 error[E0423]: expected value, found struct `String`
-  --> $DIR/issue-22692.rs:2:13
+  --> $DIR/issue-22692.rs:9:13
    |
 LL |     let _ = String.new();
-   |             ^^^^^^----
-   |             |
-   |             help: use the path separator to refer to an item: `String::new`
+   |             ^^^^^^- help: use the path separator to refer to an item: `::`
 
-error: aborting due to previous error
+error[E0423]: expected value, found struct `String`
+  --> $DIR/issue-22692.rs:13:13
+   |
+LL |     let _ = String.default;
+   |             ^^^^^^- help: use the path separator to refer to an item: `::`
+
+error[E0423]: expected value, found struct `Vec`
+  --> $DIR/issue-22692.rs:17:13
+   |
+LL |     let _ = Vec::<()>.with_capacity(1);
+   |             ^^^^^^^^^- help: use the path separator to refer to an item: `::`
+
+error[E0423]: expected value, found struct `std::cell::Cell`
+  --> $DIR/issue-22692.rs:3:9
+   |
+LL |         ::std::cell::Cell
+   |         ^^^^^^^^^^^^^^^^^
+...
+LL |     make!().get();
+   |     -------- help: use the path separator to refer to an item: `::`
```

Comment:
I suppose in the case where macros are involved, the highlights is not the greatest and `span_suggestion_verbose` could look better. On this line the `-------` for `make!()` sits directly next to the `-` for `.` and they visually blend together.